addons/actions/src/preset/addArgs.ts

```diff
@@ -0,0 +1,54 @@
+import { ParameterEnhancer, combineParameters } from '@storybook/client-api';
+import { Args } from '@storybook/addons';
+
+import { action } from '../index';
+
+// interface ActionsParameter {
+//   disable?: boolean;
+//   args?: string[];
+//   argTypesRegex?: RegExp;
+// }
+
+/**
+ * Automatically add action args for argTypes whose name
+ * matches a regex, such as `^on.*` for react-style `onClick` etc.
+ */
+export const inferActionsFromArgTypes: ParameterEnhancer = context => {
+  const { args, actions, argTypes } = context.parameters;
+  if (!actions || actions.disable || !actions.argTypesRegex || !argTypes) {
+    return null;
+  }
+
+  const argTypesRegex = new RegExp(actions.argTypesRegex);
+  const actionArgs = Object.keys(argTypes).reduce((acc, name) => {
+    if (argTypesRegex.test(name)) {
+      acc[name] = action(name);
+    }
+    return acc;
+  }, {} as Args);
+
+  return {
+    args: combineParameters(actionArgs, args),
```

Comment:
Why `combineParameters`? an action doesn't have any subfields, does it?

Reply:
Why not? Is it sufficiently slower that I should care?

Reply:
I guess not? Just sort of confusing given they are all functions.